builtin/mainmenu/dlg_settings_advanced.lua

```diff
@@ -649,11 +649,11 @@ local function create_change_setting_formspec(dialogdata)
 
 		height = height + 0.3
 		formspec = formspec
-				.. "field[0.3," .. height .. ";3.3,1;te_x;X;" -- X
+				.. "field[0.3," .. height .. ";3.3,1;te_x;"..fgettext("X")..";" -- X
 				.. core.formspec_escape(v3f[1] or "") .. "]"
-				.. "field[3.6," .. height .. ";3.3,1;te_y;Y;" -- Y
+				.. "field[3.6," .. height .. ";3.3,1;te_y;"..fgettext("Y")..";" -- Y
 				.. core.formspec_escape(v3f[2] or "") .. "]"
-				.. "field[6.9," .. height .. ";3.3,1;te_z;Z;" -- Z
+				.. "field[6.9," .. height .. ";3.3,1;te_z;"..fgettext("Z")..";" -- Z
```

Comment:
Inconsistent use of spaces around '..'
Possible overly-long line (limit is 90 with tab size 4)